data/evm-contract-map/evm-contract-map.json

```diff
@@ -970,5 +970,14 @@
     "decimals": 18,
     "coingeckoId": "nearpad",
     "chainId": "0x4e454152"
+  },
+ "0xc581b735a1688071a1746c968e0798d642ede491": {
+    "name": "Euro Tether",
+    "logo": "eurt.png",
+    "erc20": true,
+    "symbol": "EURT",
+    "decimals": 18,
+    "coingeckoId": "tether-eurt",
+    "chainId": "0x3"
```

Comment:
Are you sure the chainId is correct?

For Ethereum mainnet, this is the right place: https://github.com/brave/token-lists/blob/8af5fa2724ed130ceb936fb68b429afbe079d3b4/scripts/util.cjs#L21-L41